auth-web/src/store/modules/business.ts

```diff
@@ -154,6 +154,38 @@ export default class BusinessModule extends VuexModule {
       }
       affiliatedEntities.push(entity)
     })
+
+    const resp = await OrgService.getAffiliationInvitations(this.currentOrganization.id)
+    if (resp?.data?.affiliationInvites) {
+      const affiliationInviteInfos: AffiliationInviteInfo[] = resp.data.affiliationInvites
+
+      affiliationInviteInfos.forEach(aii => {
+        const business: Business = affiliatedEntities.find(businesses => businesses.businessIdentifier === aii.business.businessIdentifier)
+        if (business) {
+          if (business.affiliationInvites) {
```

Comment:
There's probably a better way of doing this in Javascript

Maybe possible to use chaining operator? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

I can think of how to do it in python:

```
    hash_table = {}
    hash_table.setdefault('key', []).append(1)
```

![image](https://github.com/bcgov/sbc-auth/assets/3484109/2a46a28e-6f72-4a21-9bd7-a0322e333a1c)


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment

https://stackoverflow.com/questions/14614134/create-array-and-push-into-it-in-one-line

Reply:
`(window.foo ||= []).push('bar')` <-- maybe that will help

Reply:
from GPT:
```
const resp = await OrgService.getAffiliationInvitations(this.currentOrganization.id);
const affiliationInviteInfos = resp?.data?.affiliationInvites || [];

affiliationInviteInfos.forEach(aii => {
  const business = affiliatedEntities.find(businesses => businesses.businessIdentifier === aii.business.businessIdentifier);
  const aiiArr = [aii];

  if (business) {
    business.affiliationInvites = (business.affiliationInvites || []).concat(aiiArr);
  } else {
    const b = { ...aii.business, affiliationInvites: aiiArr };
    affiliatedEntities.push(b);
  }
});
```

Reply:
Sounds good